platform/javascript/godot_shell.html

```diff
@@ -351,24 +351,7 @@
 			};
 		};
 	//]]></script>
-	<script type="text/javascript" src="$GODOT_FS"></script>
-	<script>
```

Comment:
Yes, Emscripten now inserts this piece of code in place of the `{{ SCRIPT }}` placeholder at compile-time, because we pass the `--shell-file` argument to the linker. This code was so far hard-coded, but for WebAssembly we need different code here, so we leave it to Emscripten.